charts/studio/values.yaml

```diff
@@ -309,6 +309,12 @@ postgresql:
         # -- Postgres database
         database: iterativeai
 
+ray:
+  # -- Ray enabled
+  enabled: true
+  # -- Ray name override
+  fullnameOverride: studio-ray
```

Comment:
No, let's not overwrite it, better is to relay on the `{{ .Release.Name }}`. I want to fix the this everywhere, because it does not allow to deploy the helm-chart in same namespace (e.g. local development)

Related PR: https://github.com/iterative/helm-charts/pull/120

Reply:
Agree with Oded :-) 

@0x2b3bfa0 just to confirm what we were talking: don't take fixing `postgres` or `redis` here. Just 🙏 not hardcode another thing :D in #120, I started encoding this. Maybe the @omesser hint is something I was looking for there[^1].

[^1]: I didn't spend much time on this so far, less than 15 minutes for changing and testing :D

Reply:
I'm proposing below the simplest possible alternative. I'd consider applying any further improvements to naming separately.